dist/minimal-lightbox.js

```diff
@@ -1215,7 +1216,7 @@ var Zoomable = (function () {
         }
       } else {
         var translate = getTranslate(element);
-        var scale = getZoom(element);
+        var scale = getZoom(element, this.useActualMax);
```

Comment:
Maybe, but if the class was used later these are things we don't necessarily want to expose publicly outside the class:

```js
let image = new Zoomable(document.querySelector('.some-image'));

// later...

// not sure this is something that should really be exposed in the return
image.getZoom(); 
```